README.md

```diff
@@ -153,7 +153,7 @@ Example:
 
 ```
 [
```

Comment:
We're returning an array because the endpoint supports multiple files? We should modify line 148 to indicate it can be multiple files.

> Takes a multipart file upload. Only accepts 1 field which is a file.

Reply:
We are returning an array because we can support multiple files in the future, right via this pr it only supports 1.

Reply:
Adjusted the test, maybe the new text is more clear?